src/containers/projects/empire_app/EmpireAppSections.js

```diff
@@ -7,53 +7,51 @@ import ArticleImage from '../../../assets/images/empire-app/article-image.png';
 import ExpoDemo from '../../../assets/images/empire-app/expo-demo.gif';
 import XDDemo from '../../../assets/images/empire-app/xd-demo.png';
 import { COLORS, NEUTRALS } from '../../../core/styles/Colors';
+import * as Routes from '../../../core/router/Routes';
 
 const shadow = 'drop-shadow(2px 8px 6px rgba(0, 0, 0, 0.25))';
 
 const EMPIRE_SECTIONS :Object[] = [
   {
-    name: 'section_1',
+    bgColor: NEUTRALS.WHITE,
+    buttonText: 'Check out the app',
+    description: `Empire is my go-to game for large groups because it is easy to learn and
+    works well with any number of players. The downside is someone has
+    to sit out to facilitate the game. I wanted to create an app with React Native,
+    and this seemed like a great excuse.`,
+    external: true,
+    header: 'The game',
     image: AppLogo,
     imageOnLeft: true,
     imageWidth: '25%',
-    header: 'The game',
-    description: `Empire is my go-to game for large groups because it is easy to learn and
-    works well with any number of players. The downside is that someone has
-    to sit out during the game to collect all the answers and make up fake answers.
-    I had been wanting to create an app with React Native for a while,
-    and this seemed like the perfect excuse.`,
-    bgColor: NEUTRALS.WHITE,
+    link: Routes.EMPIRE_APP_APP_STORE,
+    name: 'section_1',
```

Comment:
it feels weird to me to see lowercase snake_case being used in React, even though this is a variable's value. is there a particular reason you decided to format it this way instead of simply 'section 1'?